Extensions/Today/TodayViewController.swift

```diff
@@ -80,8 +80,29 @@ class TodayViewController: UIViewController, NCWidgetProviding {
         return button
     }()
 
-    private lazy var buttonSpacer: UIView = UIView()
-    private var heightConstraint: Constraint?
+    private var widgetStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .Vertical
+        stackView.alignment = .Fill
+        stackView.spacing = 0
+        stackView.distribution = UIStackViewDistribution.Fill
+        stackView.layoutMargins = UIEdgeInsets(top: TodayUX.margin, left: TodayUX.margin, bottom: TodayUX.margin, right: TodayUX.margin)
+        stackView.layoutMarginsRelativeArrangement = true
+        return stackView
+    }()
+    
+    private lazy var buttonStackView: UIStackView = {
```

Comment:
Nit: This is `lazy`, but `widgetStackView` isn't. For the sake of consistency, I'd make them both `lazy` or non-`lazy` (don't think it matters much either way).